src/vs/workbench/contrib/notebook/browser/viewParts/notebookKernelView.ts

```diff
@@ -32,13 +32,30 @@ function getEditorFromContext(editorService: IEditorService, context?: KernelQui
 		editor = getNotebookEditorFromEditorPane(matchingEditor);
 	} else if (context !== undefined && 'notebookEditor' in context) {
 		editor = context?.notebookEditor;
+	} else if (context !== undefined && 'notebookUri' in context) {
+		const matchingEditor = editorService.visibleEditorPanes.find((editor) => {
```

Comment:
I could make this more robust by also checking the viewColumn, or only checking the activeEditor, but it feels pretty clunky, and I couldn't figure out a better way to pass the editor info from an extension